### PR TITLE
Fix ios sideframe scrolling

### PR DESCRIPTION
**Pull request by vxsx**
Glitches on iPads were amazing, the fix is not, but it works.

### CODE REVIEW

File: cms/static/cms/js/modules/cms.sideframe.js [hunk 1]
```diff
@@ -215,6 +215,37 @@ var CMS = window.CMS || {};
...
+                 * (this is an iOS glitch, the content would still be there and in fact it would
+                 * be usable, but just not visible). To get rid of that we bring up the shim element
+                 * up and down again and this fixes the glitch. (same shim we use for resizing the sideframe)
+                 *
+                 * It is not recommended to expose it and use it on other devices rather than iOS ones.
+                 *
+                 * @function forceRerenderOnIOS
+                 * @private
+                 */
+                function forceRerenderOnIOS() {
```
Comment (FinalAngel) [inline cms/static/cms/js/modules/cms.sideframe.js]: beautiful :name_badge:

**Comment by FinalAngel** [comment]
LGTM...

**Comment by lory87** [comment]
:100:

**Comment by Brankub** [comment]
:dancers: on :meat_on_bone:

**Pull request merged**